app/helpers/alert_notifier_helper.rb

```diff
@@ -60,4 +60,16 @@ def new_subscription_url_with_tracking(alert: nil, utm_content: nil)
 
     new_subscription_url(params)
   end
+
+  def subject(alert, applications, comments)
+    items = if !applications.empty? && comments.empty?
+      pluralize(applications.size, "new planning application")
+    elsif applications.empty? && !comments.empty?
+      pluralize(comments.size, "new comment") + " on planning applications"
+    elsif !applications.empty? && !comments.empty?
+      pluralize(comments.size, "new comment") + " and " + pluralize(applications.size, "new planning application")
+    end
+
+    "#{"You're missing out on: " if alert.expired_subscription?}#{items} near #{alert.address}"
```

Comment:
While this is succinct on one line, the most common case that people will see when there's no subscription is squashed into the far conner. Even though there's repetition, would something like this be simpler:

```
if alert.expired_subscription?
  "You’re missing out on #{items} near #{alert.address}"
else
  "#{items} near #{alert.address}"
end
```

You could even take out the repetition with a `sentence_base` variable or something. I think that's a bit easier to read, but if you think this is clearer :+1: